### PR TITLE
AT_04.14.02 The Week button is located in the upper right corner of the section Departure date, visible and have text "Week"

### PR DESCRIPTION
**Pull request by Albina-star**
TC - https://trello.com/c/fIbUqquR/962-tc041402-week-button-elements-view-the-week-button-is-located-in-the-upper-right-corner-of-the-section-departure-date

AT - https://trello.com/c/hhwrb8J2/961-at041402-week-button-elements-view-the-week-button-is-located-in-the-upper-right-corner-of-the-section-departure-date

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.14_Week button elements view UI.cy.js [hunk 1]
```diff
@@ -26,4 +26,9 @@ describe('US_04.14 | Week button elements view ', () => {
     it('AT_04.14.01 | In the upper left corner of the section there is a label Departure date which has the color #666666', function() {
         createBookingPage.getLableDepartureDate().should('have.css','color', this.colors.greyHeader)
     });
+
+    it('AT_04.14.02| The Week button is located in the upper right corner of the section Departure date', function () {
+        createBookingPage.getWeekButton().should('be.visible');
+        createBookingPage.getWeekButton().should('have.text', this.createBookingPage.weekButtonText);
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/US_04.14_Week button elements view UI.cy.js:32]: The second assertion is better done through `and`, so as not to duplicate the code:

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/createBookingPage/US_04.14_Week button elements view UI.cy.js (hunk 1 above)
Comment (Albina-star) [reply cypress/e2e/testUiAndFunction/createBookingPage/US_04.14_Week button elements view UI.cy.js]: Corrected. Thank you Roman for review.

**Review by OLgaByR — COMMENTED** [review]
Как Вы считаете, стоит ли переназвать тест, вы же не проверяете локацию эл в тесте, вверху или в каком углу, а протсо проверяете тест и видимость?

**Review by OLgaByR — COMMENTED** [review]
"The Week button is visible and have text "Week"  - we dont check in test if week button is located in the upper right corner of the section Departure date

**Review by EkaterinaRomankova — APPROVED** [review]

**Pull request merged**